src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
       case f: WdlFile =>
-        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.toInt)
+        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.trim.toInt)
```

Comment:
Hmm. It doesn't look like that's changed from a time back when I know this was the case as people used to bitch about it on the tubez and important ppl would say "so don't use fromFile, silly!" *shrug*